worker/process_batch_worker.go

```diff
@@ -282,14 +282,25 @@ func (b *ProcessBatchWorker) Process(message *workers.Msg) {
 	}
 	log.D(l, "Sent push to pusher for batch users.")
 	err = b.updateJobBatchesInfo(parsed.JobID)
-	checkErr(l, err)
+	b.checkErr(job, err)
 	log.D(l, "Updated job batches info successfully.")
 	err = b.updateJobUsersInfo(parsed.JobID, len(parsed.Users)-batchErrorCounter)
-	checkErr(l, err)
+	b.checkErr(job, err)
 	log.D(l, "Updated job users info successfully.")
 	if float64(batchErrorCounter)/float64(len(parsed.Users)) > b.Workers.Config.GetFloat64("workers.processBatch.maxUserFailureInBatch") {
-		b.incrFailedBatches(job.ID, job.TotalBatches, parsed.AppName)
-		checkErr(l, fmt.Errorf("failed to send message to several users, considering batch as failed"))
+		b.incrFailedBatches(job, parsed.AppName)
+		b.checkErr(job, fmt.Errorf("failed to send message to several users, considering batch as failed"))
 	}
+
+	b.Workers.Statsd.Incr(processBatchWorkerCompleted, job.Labels(), 1)
```

Comment:
Should we not cover all these returns?
* https://github.com/topfreegames/marathon/pull/51/files#diff-77f6b43c0608b2514ab8d6038ce06ec4e1961bd168589954a328405d9e1881cbR185
* https://github.com/topfreegames/marathon/pull/51/files#diff-77f6b43c0608b2514ab8d6038ce06ec4e1961bd168589954a328405d9e1881cbR189-R199